src/gateways/mappers/beaconResponse.ts

```diff
@@ -0,0 +1,36 @@
+import { IApiResponse } from "./apiResponse";
+
+export interface IBeaconResponse extends IApiResponse {
```

Comment:
@tm1303 Should this be `IBeaconListResponse`? 

In the future I imagine we are going to need to hit the get beacon by id endpoint which will return a single instance of this, i.e: 

```typescript
export interface IBeaconResponse extends IApiResponse {
  data: IBeaconDataAttributes
```